File: 240115/코드네임/code-name.java
import java.util.Scanner;
class student{
    char alpha;
    int num;

    public student(char alpha,int num){
        this.alpha = alpha;
        this.num = num;
    }
}
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);

        student[] exam = new student[5];

        for(int i = 0 ; i<5; i++){
            char alpha = sc.next().charAt(0);
            int num = sc.nextInt();

            exam[i] = new student(alpha,num); 
        }

        int mindex = 0 ;

        for(int i = 1 ; i<5; i++){
            if (exam[mindex].num > exam[i].num){
                mindex = i;
            }
        }
        System.out.print(exam[mindex].alpha+" "+exam[mindex].num);
    }
}

File: 240115/코드네임/code-name.py
class exam:

    def __init__(self, grade, num):
        self.grade = grade
        self.num = num
student = []
for _ in range(5):
    grade,num = tuple(input().split())
    student.append(exam(grade,int(num)))

min_val = 0 

for i in range(1,5):
    if student[min_val].num > student[i].num:
        min_val = i

print(student[min_val].grade,student[min_val].num)